ui/src/app/dashboard/dashboard.component.css

```diff
@@ -0,0 +1,23 @@
+* {
+  font-family: "Gotham HTF", sans-serif;
+}
+
+div {
+  margin: 1rem;
+}
+
+.aggregation-container {
+  padding: 1rem;
+}
+
+.group-container {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
FYI: It's unfortunate that expanding a card bumps the other contents down - I toyed around with CSS grid, but this will probably a bit more complicated than we'd want right now, and I still don't love how it reshuffles the other cards.